sources/presentation/Stride.Core.Presentation.Quantum/ViewModels/GraphViewModel.cs

```diff
@@ -41,7 +41,6 @@ private GraphViewModel([NotNull] IViewModelServiceProvider serviceProvider, [Not
             GraphViewModelService = serviceProvider.TryGet<GraphViewModelService>();
             if (GraphViewModelService == null) throw new InvalidOperationException($"{nameof(GraphViewModel)} requires a {nameof(GraphViewModelService)} in the service provider.");
             Logger = GlobalLogger.GetLogger(DefaultLoggerName);
-            if (rootPresenters == null) throw new ArgumentNullException(nameof(rootNode));
```

Comment:
I removed this line entirely instead of implementing the suggested change as `rootNode` is supposed to be null at this point.